bindings/stronghold-nodejs/examples/src/tests/multiple_identities.ts

```diff
@@ -4,6 +4,7 @@ import { stronghold } from '../stronghold';
 // Only verifies that no uncaught exceptions are thrown, including syntax errors etc.
 describe("Test Stronghold Node.js examples", function () {
     it("Multiple Identities", async () => {
-        await multipleIdentities(await stronghold());
+        // TODO: Temporarily disabled until iotaledger/stronghold.rs#353 is fixed.
```

Comment:
should be okay again after a merging `dev`

Reply:
To clarify, it's disabled on `dev` too.

Reply:
Can we use an async `RwLock` over the Stronghold storage implementation to prevent this bug in the meantime then?

Reply:
I guess we can, but this issue is supposed to be prioritized now, so perhaps @felsweg-iota can give an update on the current status.

Reply:
The current problem with the implementation of the standard interface to Stronghold is, that it internally uses a std::sync::RwLock, since every function in the interface isn't supposed to be `async`. This actually brings in a problem while working with `async` code. 

There are two solutions to this, and i am just naming them here for reference, @PhilippGackstatter already pointed out the link to the issue: 

- we either change the current standard library locking mechanism to their `async` counterparts, which will effectively change the interface to operate as pseudo-`async`, 
- the STM based approach will be finalized and will replace all existing locks.

Reply:
I think I'd prefer if we try the `RwLock` solution in our code to fix it temporarily (if that works), and that could avoid you having to do temporary work if the STM will eventually be the proper solution.